### ISSUE TITLE
wcag22aa tag showing No violations

### ISSUE THREAD

**Issue by sethims** [dequelabs/axe-core#4560]
### Product

axe-core

### Product Version

4.10.0

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
I am trying to use the tag wcag22aa to test accessibility violations against WCAG 2.2 AA standard.
#### Actual
My Test case is getting passed with No Violation Found

#### How to Reproduce
using below two lines after visiting the screen where I need to verify accessibility-
 cy.injectAxe();
 cy.checkA11y(null, {runOnly: {type: "tag",values: ["wcag22aa"]}});

**Comment by straker** [comment]
Thanks for the issue. It looks like you're trying to use the [cypress-axe](https://github.com/component-driven/cypress-axe) package, which isn't maintained by us. You'll need to open an issue with them.
